keylime/tpm/tpm_main.py

```diff
@@ -762,7 +763,7 @@ def activate_identity(self, keyblob: bytes) -> Optional[bytes]:
                 command = [
                     "tpm2_activatecredential",
                     "-c",
-                    aik_keyhandle,
+                    cast(str, aik_keyhandle),
```

Comment:
That seems unsafe, from one side we cast this variable as `int` and here as `str`. Cast does not change the type.

We need to agree on what we expect for `aik_keyhandle`, that seems to be `int` (there is a call to hex, that only works as if it is this type). That way we can say here `str(aik_keyhandle)` with confidence (and no casting).

Reply:
aik_handle needs to go onto the command line as a hex number for older tools and is a path to a file (iirc) on recent versions of the tools. I am going to replace this with int() and str() since it can be both. It may be also worth considering removal of support for older versions of the tools.